### ISSUE TITLE
Grid: Title attributes on cells incorrectly used (1.3.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#940]
### Expected behavior
Title attributes are used for advisory information related to the element. It shouldn't repeat the same information as the table cell. There shouldn't be any title attributes with same values.

### Actual behavior
The title values in cells are the same as the text value of the cell. This is problematic for some screenreaders as they will read out both. For a user listening a 1 in a cell would be read as 1 1  which could be interpreted as 11 instead of the true value of 1. For text values, it's just annoying to hear them twice.
